modules/analytics-v2/src/views/full/index.tsx

```diff
@@ -14,14 +17,23 @@ const COLOR_MESSENGER = '#0196FF'
 const COLOR_WEB = '#FFA83A'
 const COLOR_TELEGRAM = '#2EA6DA'
 const SECONDS_PER_DAY = 86400
+const sideList = [
+  { key: 'dashboard', label: 'Dashboard' },
+  { key: 'agentUsage', label: 'Agent Usage' },
+  { key: 'engagement', label: 'Engagement & Retention' },
+  { key: 'understanding', label: 'Understanding' }
+]
 
 export default class Analytics extends React.Component<{ bp: any }> {
```

Comment:
Can you convert it as a react hook instead of a class ? We try to do the conversion when we have to touch an existing class component (plus.. this is new development)